README.md

```diff
@@ -11,6 +11,8 @@ REACT_APP_API_HOST=http://localhost:3000
 PORT=3112
 REACT_APP_SHOW_NOTIFICATIONS=true
 REACT_APP_AUTHENTICATION_HOST=http://localhost:3001
+REACT_APP_H5P_ASSETS_HOST=https://graasp-h5p.s3.eu-central-1.amazonaws.com
+REACT_APP_H5P_CONTENT_HOST=http://localhost:3000/p
```

Comment:
Should we change this to something more specific? Or is it supposed to be the `p` of public?